lib/board.rb

```diff
@@ -0,0 +1,24 @@
+class Board
+  attr_accessor :positions
+
+  def initialize
+    @positions = Array.new(6) { Array.new(7, nil) }
+  end
+
+  def display_board
```

Comment:
Don't `puts` and `gets` all over your logic. putting and getting is basically your interaction with the window. 
For simplicity, let's say you are only allowed to do `puts` and `gets` in your index file. 
This method can return a printable version of it's data, and as such, should be called more accurately as `to_s`, `print`, or maybe `render` ?